src/templates/button-edit.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActionButton from './action-button';
+import { keys } from '../helpers/input';
+import { linkProps } from './functions';
+
+const openComment = note => () => {
+  const { 'data-comment-id': commentId, 'data-site-id': siteId } = linkProps(note);
+  window.open(`https://wordpress.com/comment/${siteId}/${commentId}`);
```

Comment:
is this what we really want here? did we consider firing an action and handling the open in middleware?

Reply:
#177 is a good example of that.

However let's also be careful to respect options.
- wpcom should go to calypso comment management but Jetpack folks who did have `edit_links_calypso_redirect` set to false needs to visit wp-admin

Reply:
Heads up that we'll want to update Calypso handlers before bumping notifications there. cc @kwight

Reply:
> is this what we really want here? did we consider firing an action and handling the open in middleware?

I see, it makes sense. I'll follow #177 and update this.

> wpcom should go to calypso comment management but Jetpack folks who have `edit_links_calypso_redirect` set to false needs to visit wp-admin

I completely forgot about that! 🤦‍♂️ 

> Heads up that we'll want to update Calypso handlers

Not sure I got this.

Reply:
> Not sure I got this.

The standalone handlers are for the iframe version (used in masterbar/mc). If I'm not mistaken, we'll need action handlers updated in Calypso too.

Reply:
And we also need to update the Masterbar JS to track the new event on WordPress.com and Jetpack. I've added some code here: https://github.com/Automattic/notifications-panel/pull/222#discussion_r165087279